ts/sagas/profile.ts

```diff
@@ -63,8 +63,47 @@ import {
   getLocalePrimaryWithFallback
 } from "../utils/locale";
 import { readablePrivacyReport } from "../utils/reporters";
+import { LollipopConfig } from "./../features/lollipop";
 
 // A saga to load the Profile.
+export function* loadProfileLollipop(
+  lollipopConfig: LollipopConfig,
+  getProfileLollipop: ReturnType<typeof BackendClient>["getProfileLollipop"]
+): Generator<
+  ReduxSagaEffect,
+  O.Option<InitializedProfile>,
+  SagaCallReturnType<typeof getProfileLollipop>
+> {
+  try {
+    const response = yield* call(getProfileLollipop, lollipopConfig);
+    // we got an error, throw it
```

Comment:
```suggestion
```

Reply:
This is a legacy comment, I'll leave it here for a future refactoring.